### PR TITLE
Fix image output

### PR DESCRIPTION
**Pull request by fahedouch**
avoid creating digest images for each manifest in the index.

fixing https://github.com/containerd/nerdctl/issues/177

### CODE REVIEW

File: cmd/nerdctl/load.go [hunk 1]
```diff
@@ -66,7 +65,7 @@ func loadImage(in io.Reader, clicontext *cli.Context) error {
 	defer cancel()
 
 	sn := clicontext.String("snapshotter")
-	imgs, err := client.Import(ctx, in, containerd.WithDigestRef(archive.DigestTranslator(sn)), containerd.WithAllPlatforms(clicontext.Bool("all-platforms")))
+	imgs, err := client.Import(ctx, in, containerd.WithAllPlatforms(clicontext.Bool("all-platforms")))
```
Comment (ktock) [inline cmd/nerdctl/load.go:68]: This change makes all non-tagged images loaded by `nerdctl load` get GC-ed immediately.

Example:
```
nerdctl pull busybox:latest
nerdctl save busybox:latest > /tmp/busybox.tar
mkdir /tmp/busybox
tar -C /tmp/busybox -xf /tmp/busybox.tar
jq 'del(.manifests[].annotations)' /tmp/busybox/index.json > /tmp/busybox/index.json.back
mv /tmp/busybox/index.json.back /tmp/busybox/index.json
jq 'del(.[].RepoTags)' /tmp/busybox/manifest.json > /tmp/busybox/manifest.json.back
mv /tmp/busybox/manifest.json.back /tmp/busybox/manifest.json
(...clear containerd cache...)
tar -C /tmp/busybox/ -c . | nerdctl load
(no image loaded here)
```

**Review by ktock — CHANGES_REQUESTED** [review]

**Comment by ktock** [comment]
I believe https://github.com/containerd/containerd/pull/5997 will be needed for solve it. PTAL.

**Comment by AkihiroSuda** [comment]
> containerd/containerd#5997

Merged

**Comment by fahedouch** [comment]
thank you @AkihiroSuda , waiting for next containerd release to update the PR.

**Comment by ktock** [comment]
> waiting for next containerd release to update the PR.

@fahedouch Do we need to wait for the next release? That patch changes only client library so we can work on it with the main branch of containerd.

**Comment by AkihiroSuda** [comment]
@ktock Would you be interested in carrying this PR?

**Comment by fahedouch** [comment]
@AkihiroSuda @ktock  
if you allow, I can take care of it

**Comment by AkihiroSuda** [comment]
Thanks @fahedouch , yes, please

**Comment by ktock** [comment]
Thanks @fahedouch !

**Comment by fahedouch** [comment]
@ktock @AkihiroSuda  is it good ?

**Review by ktock — APPROVED** [review]

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**